plugins/MauticCrmBundle/Integration/SugarcrmIntegration.php

```diff
@@ -194,18 +212,41 @@ public function getFormCompanyFields($settings = [])
     }
 
     /**
-     * @param array $settings
+     * Get available fields for choices in the config UI.
      *
-     * @return array|mixed
+     * @param array $settings
      *
-     * @throws \Exception
+     * @return array
      */
     public function getFormLeadFields($settings = [])
     {
-        $leadFields    = $this->getFormFieldsByObject('Leads', $settings);
-        $contactFields = $this->getFormFieldsByObject('Contacts', $settings);
+        if (!$this->isAuthorized()) {
+            return [];
+        }
+
```

Comment:
@kuzmany could you add a `if isset` here to prevent a notice and warning if value is not set?

```php
[2018-05-28 10:08:03] mautic.NOTICE: PHP Notice - Undefined index: feature_settings - in file /home/path/plugins/MauticCrmBundle/Integration/SugarcrmIntegration.php - at line 229 [] []
>[2018-05-28 10:08:03] mautic.WARNING: PHP Warning - array_combine() expects parameter 1 to be array, null given - in file /home/path/plugins/MauticCrmBundle/Integration/SugarcrmIntegration.php - at line 229 [] []
```